src/ai/backend/common/web/session/redis_storage.py

```diff
@@ -20,13 +20,18 @@
 class RedisStorage(AbstractStorage):
     """Redis storage"""
 
+    @property
+    def login_session_extend_time(self) -> Optional[int]:
+        return self._login_session_extend_time
```

Comment:
This name is confusing. It is hard to understand that this variable is used to be as a flag that allows to update expiration time or is used as a default value to extend the expiration time or a default value of max age of sessions.
Let's give a better name to it

Reply:
Good point. The usage of this values is to store the amount of expiration time of login session cookie. How about login_session_extend_time_in_seconds ?